packages/block-editor/src/components/block-toolbar/use-has-block-toolbar.js

```diff
@@ -21,11 +21,10 @@ export function useHasBlockToolbar() {
 			const {
 				getBlockEditingMode,
 				getBlockName,
-				getSelectedBlockClientIds,
+				getBlockSelectionStart,
 			} = select( blockEditorStore );
 
-			const selectedBlockClientIds = getSelectedBlockClientIds();
-			const selectedBlockClientId = selectedBlockClientIds[ 0 ];
+			const selectedBlockClientId = getBlockSelectionStart();
```

Comment:
Do you think adding a comment of why this is used (perf) instead of the selector that has the name matching the variable name is something future us will like?

Reply:
I don't think someone would want to change this unless they knew why they were changing it, so maybe it's OK without the comment.